src/Impostor.Api/Config/AntiCheatConfig.cs

```diff
@@ -6,6 +6,22 @@ public class AntiCheatConfig
 
         public bool Enabled { get; set; } = true;
 
+        public bool ForbidHostCheating { get; set; } = true;
+
+        public bool ForbidProtocolExtensions { get; set; } = true;
+
+        public bool EnableGameFlowChecks { get; set; } = true;
+
+        public bool EnableMustBeHostChecks { get; set; } = true;
+
+        public bool EnableLimitChecks { get; set; } = true;
+
+        public bool EnableOwnershipChecks { get; set; } = true;
+
+        public bool EnableRoleChecks { get; set; } = true;
+
+        public bool EnableTargetChecks { get; set; } = true;
```

Comment:
Did you consider making a `Dictionary<CheatCategory, bool> EnabledCategories` or something like that instead?

Reply:
No, I didn't know how configs could be nested. A dictionary sounds annoying to set via

Reply:
No, I didn't know how configs could be nested. A dictionary sounds annoying to set via environment variables